### ISSUE TITLE
[CI] Tests in get_autoscaling_capacity fail

### ISSUE THREAD

**Issue by dimitris-athanasiou** [elastic/elasticsearch#64795]
2 tests failing in `org.elasticsearch.xpack.autoscaling.AutoscalingRestIT`: 

- test {p0=autoscaling/get_autoscaling_capacity/Test get empty autoscaling capacity}
- test {p0=autoscaling/get_autoscaling_capacity/Test get fixed autoscaling capacity}

**Build scan**: https://gradle-enterprise.elastic.co/s/jqr4bshzg2ex6

**Repro line**:  ```./gradlew ':x-pack:plugin:autoscaling:qa:rest:yamlRestTest' --tests "org.elasticsearch.xpack.autoscaling.AutoscalingRestIT.test {p0=autoscaling/get_autoscaling_capacity/Test get fixed autoscaling capacity}" -Dtests.seed=BAF47F5BCC8D4F92 -Dtests.security.manager=true -Dtests.locale=ar-LB -Dtests.timezone=Pacific/Guam -Druntime.java=11```

**Reproduces locally?**: No

**Applicable branches**: master

**Failure history**:

A few failures recently.

https://build-stats.elastic.co/app/kibana#/discover?_g=(refreshInterval:(pause:!t,value:0),time:(from:now-60d,mode:quick,to:now))&_a=(columns:!(_source),index:b646ed00-7efc-11e8-bf69-63c8ef516157,interval:auto,query:(language:lucene,query:get_autoscaling_capacity),sort:!(process.time-start,desc))

**Failure excerpt**:

```

[2020-11-10T00:52:24,436][INFO ][o.e.x.a.AutoscalingRestIT] [test] [p0=autoscaling/get_autoscaling_capacity/Test get fixed autoscaling capacity] before test |  
-- | --
  | [2020-11-10T00:52:24,517][INFO ][o.e.x.a.AutoscalingRestIT] [test] Stash dump on test failure [{ |  
  | "stash" : { |  
  | "body" : { |  
  | "error" : { |  
  | "root_cause" : [ |  
  | { |  
  | "type" : "illegal_argument_exception", |  
  | "reason" : "results can not be empty", |  
  | "stack_trace" : "org.elasticsearch.ElasticsearchException$1: results can not be empty |  
  | at org.elasticsearch.ElasticsearchException.guessRootCauses(ElasticsearchException.java:644) |  
  | at org.elasticsearch.ElasticsearchException.generateFailureXContent(ElasticsearchException.java:572) |  
  | at org.elasticsearch.rest.BytesRestResponse.build(BytesRestResponse.java:149) |  
  | at org.elasticsearch.rest.BytesRestResponse.<init>(BytesRestResponse.java:110) |  
  | at org.elasticsearch.rest.BytesRestResponse.<init>(BytesRestResponse.java:93)


```

**Comment by henningandersen** [comment]
Opened #64796

**Comment by henningandersen** [comment]
Merged #64796 so closing this.
